lib/checkWin.ts

```diff
@@ -41,16 +45,24 @@ const checkForDraw = async (game) => {
   }
 };
 
-const checkForWin = async (game) => {
+const checkForWin = async (game: Game): Promise<boolean> => {
   let winner = await checkMappedGridForWin(game, mapGridValues(game));
   return Boolean(winner);
 };
 
-const logGameOver = async (game, win, draw) => {
+const logGameOver = async (
+  game: Game,
+  win: boolean,
+  draw: boolean
+): Promise<void> => {
   if (win || draw) {
-    let log;
+    let log: string;
     let gameOverObj;
     if (win) {
+      if (!game.winner) {
```

Comment:
- Set game.winnner to be of type Team | null
- Assign game.winner to a variable to help with type narrowing